crates/typst/src/math/equation.rs

```diff
@@ -78,18 +81,20 @@ pub struct EquationElem {
 
     /// The alignment of the equation numbering.
     ///
-    /// By default, the number is put at the `{end}` of the equation block. Both
-    /// `{start}` and `{end}` respects text direction; for absolute positioning,
-    /// use `{left}` or `{right}`.
+    /// By default, the alignment is `{end} + {horizon}`. For the horizontal
```

Comment:
```suggestion
    /// By default, the alignment is `{end + horizon}`. For the horizontal
```
The brackets around just indicate which kind of syntax highlighting to use:
- `{...}` is `...` with `typc` syntax highlighting
- `[...]` is `...` with `typ` syntax highlighting